Added help.md (#64)
PyDrocsid/documentation: docs/cogs/information/help.md

# Help

> This cog provides diverse help commands

Reviewer: ```suggestion
This cog provides diverse help commands.
```

> ### General Usage

Reviewer: ```suggestion
### General usage
```

```
.help [cog|command]
```

> ## General Help

Reviewer: ```suggestion
## General help
```

> All commands, sorted by their respective cogs, you can use `.help`

Reviewer: ```suggestion
You can use `.help` to get a list of all commands grouped by cogs.
```

## Cogs
To list all commands of any cog, run `.help <cog>`

> ## Specific Commands

Reviewer: ```suggestion
## Specific commands
```

> A detailed description is provided by executin `.help <command>`. All aliases and usages are shown herecephox-patch-1

Reviewer: ```suggestion
A detailed description is provided by executing `.help <command>`. All aliases and usages are shown here.
```
